core-services/prow/02_config/shiftstack/prune/_pluginconfig.yaml

```diff
@@ -0,0 +1,45 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - shiftstack/prune
+  require_self_approval: false
+external_plugins:
```

Comment:
Also, there could be endpoints we need to add for the jira plugin.
I'm thinking about https://github.com/openshift/release/blob/b669424/core-services/prow/02_config/shiftstack/gazelle/_pluginconfig.yaml#L27-L31